packs/mssql/actions/lib/mssql_action.py

```diff
@@ -0,0 +1,32 @@
+import _mssql
+
+from st2actions.runners.pythonrunner import Action
+
+__all__ = [
+    'MSSQLAction'
+]
+
+
+class MSSQLAction(Action):
+
+    def __init__(self, config):
```

Comment:
You don't actually need __init__ method here, __init__ method of the parent class already does that so you can simply get rid of it here.